src/dmtcp_restart.cpp

```diff
@@ -461,7 +483,7 @@ runMtcpRestart(int fd, ProcessInfo *pInfo)
   mtcpArgs.push_back((char *) fdBuf.c_str());
 
   mtcpArgs.push_back(NULL);
-  execvpe(mtcpArgs[0], &mtcpArgs[0], environ);
+  execvp(mtcpArgs[0], &mtcpArgs[0]);
```

Comment:
I see that codefactor.io warned about including an untested args (or untested environment variables) in the execvp.  I don't think we have to worry.  We could be influenced by the user injecting DMTCP_RESTART_PAUSE, and the DEBUG variant.  But that is part of the intended design.  So, I'm not worried here.